src/test/java/com/wojciechbarwinski/demo/epic_board_games_shop/mappers/MapperFacadeTest.java

```diff
@@ -0,0 +1,181 @@
+
+package com.wojciechbarwinski.demo.epic_board_games_shop.mappers;
+
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.AddressDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderLineDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderRequestDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.dtos.OrderResponseDTO;
+import com.wojciechbarwinski.demo.epic_board_games_shop.entities.*;
+import com.wojciechbarwinski.demo.epic_board_games_shop.exceptions.MappingToDTOException;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MapperFacadeTest {
+
+    MapperFacade mapperFacade = new MapperFacade(new OrderMapper());
+
+    @Test
+    void shouldMapOrderDTOToOrderEntity(){
+        //given
+        AddressDTO addressDTO = new AddressDTO("Ulica", "Miasto", "12-345", "123-456-789");
+        OrderRequestDTO orderRequestDTO = new OrderRequestDTO("orderer mail",
+                addressDTO,
+                List.of(new OrderLineDTO(1L, 2)));
+
+        //when
+        Order order = mapperFacade.mapOrderRequestDTOToOrderEntity(orderRequestDTO);
+
+        //then
+        assertNotNull(order);
+        assertNull(order.getId()); //mapper should not map ID because don't have it
```

Comment:
I tutaj zauważyłęm, że to DTO powinno nazywać się w takim wypadku CreateOrderRequestDto